frontend/src/lib/shipment/shipmentPackagingPdf.ts

```diff
@@ -0,0 +1,158 @@
+/*
+This file is part of the SoLawi Bedarf app
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import {
+  Depot,
+  ProductCategoryWithProducts,
+  ProductsById,
+  Shipment,
+} from "../../../../shared/src/types";
+import { getLangUnit } from "../../lang/template";
+import { multiplicatorOptions } from "../options";
+import { format } from "date-fns";
+import { createDefaultPdf, PdfSpec, PdfTable } from "../pdf/pdf";
+import { sanitizeFileName } from "../../../../shared/src/util/fileHelper";
+import { Zip } from "../pdf/zip";
+import {
+  byKey,
+  findDepotNameById,
+  getOrCompute,
+  inLocaleOrder,
+} from "../utils.ts";
+
+type ProductRow = [string, string, string];
+type GroupedProducts = Map<string, ProductRow[]>;
+type DepotGroupedProducts = Map<string, GroupedProducts>;
+
+function joinStrings(...strings: (string | null | undefined)[]): string {
+  return strings.filter((s) => !!s).join(" ");
+}
+
+export function createShipmentPackagingPdfSpecs(
```

Comment:
@R1tschY, nur mal eine grundlegende Frage: ich bin in der JavaScript-Welt nicht so wirklich zu Hause, aber es gibt ja verschiedene Varianten Funktionen zu definieren: `function abc() {..}` vs. `const abc = () => {..}`

An den meisten Stellen wurde bisher die const-Variante verwendet. Hat es einen bestimmten Grund, dass du hier das function-Schlüsselwort verwendest?